divide.py

```diff
@@ -0,0 +1,2 @@
+def divide(x, z):
```

Comment:
```suggestion
def divide(x, y):
```
I think z should be y here.